### PR TITLE
Update yearly-schedule.md

### PR DESCRIPTION
**Pull request by demkodo**
Updated the yearly schedule to accurately reflect the national holidays for 2021

-----
[View rendered organization/yearly-schedule.md](https://github.com/publiccodenet/about/blob/update-yearly-schedule/organization/yearly-schedule.md)

### CODE REVIEW

File: organization/yearly-schedule.md [hunk 1]
```diff
@@ -12,29 +12,23 @@ This schedule shows recurring events the Foundation for Public Code prepares for
...
-5 May: Liberation day (public holiday in the Netherlands, office closed)
+5 May: Liberation day (public holiday in the Netherlands every five years. Next time is 2025, office closed)
 
-21 May: Ascension day (public holiday in the Netherlands, office closed)
+13 May: Ascension day (public holiday in the Netherlands, office closed)
 
 ## June
 
-31 May-1 June: Pentecost and Whit Monday (public holidays in the Netherlands, office closed)
+23 - 24 May: Pentecost and Whit Monday (public holidays in the Netherlands, office closed)
```
Comment (ElenaFdR) [inline organization/yearly-schedule.md:31]: This needs to go up to the May subheading.

**Review by ElenaFdR — COMMENTED** [review]
Ready to go, barring 1 change!

File: organization/yearly-schedule.md [hunk 2]
```diff
@@ -12,29 +12,23 @@ This schedule shows recurring events the Foundation for Public Code prepares for
...
 ## March
 
-11-13 March: [Code for America Summit](https://www.codeforamerica.org/summit) - see you there!
-
 ## April
 
-2 April: Foundation for Public Code General Assembly
-
-13 April: Easter Monday (public holiday in the Netherlands, office closed)
+5 April: Easter Monday (public holiday in the Netherlands, office closed)
```
Comment (ElenaFdR) [inline organization/yearly-schedule.md:19]: Perhaps add that we're closed on Tuesday too?

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
Useful updates. I've made some suggestions for better accuracy.

File: organization/yearly-schedule.md (hunk 2 above)
Comment (demkodo) [reply organization/yearly-schedule.md:19]: It's a exception / one-off so might be confusing. I'd prefer to leave it like this.

**Review by bvhme — APPROVED** [review]

**Pull request merged**